docs/cog_guides/cog_manager_ui.rst

```diff
@@ -0,0 +1,244 @@
+.. _cogmanagerui:
+
+==============
+Cog Manager UI
+==============
+
+This is the cog guide for the core cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: This cog is not like the other cogs. It is loaded by default, not
+    included in the cogs paths and it cannot be unloaded. It contains needed
+    commands for cog management.
+
+.. _cogmanaerui-usage:
+
+-----
+Usage
+-----
+
+This cog allows you to manage your cogs and where you can install them. Unlike
+V2, which had a ``cogs`` folder where everything was installed, you can
+install V3 cogs everywhere, and also make them cross-compatible with other
+instances!
+
+The most basic command is :ref:`paths <cogmanagerui-command-paths>`, which
+will list you all of the currently set paths.
+
+You can add a path by using the :ref:`addpath <cogmanagerui-command-addpath>`
+command. All cogs in that path will be available for the bot and listed in
+the :ref:`cogs <cogmanagerui-command-cogs>`. You can then load
+or unload them.
```

Comment:
I feel like it might be helpful to clarify that this is **only** for installing cogs from files, and that 3rd party cogs found on cog lists **do not** need to be installed this way, and should be installed with downloader instead.